.github/workflows/black.yml

```diff
@@ -0,0 +1,50 @@
+name: Black
+
+on:
+  workflow_dispatch: # Activate this workflow manually
+  pull_request:
+
+jobs:
+
+  check:
+    runs-on: ubuntu-latest
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: ./.github/actions/python_cache/
+
+      - name: Install Haystack
+        run: |
+          pip install --upgrade pip
+          pip install .[dev]
+
+      - name: Black
+        run: black .
```

Comment:
do we still need this step with the `black . --check` below?

Reply:
Oh right! No we don't :+1: